Formula/s-lang.rb

```diff
@@ -2,7 +2,7 @@ class SLang < Formula
   desc "Library for creating multi-platform software"
   homepage "http://www.jedsoft.org/slang/"
   url "http://www.jedsoft.org/releases/slang/slang-2.3.1a.tar.bz2"
-  mirror "http://pkgs.fedoraproject.org/repo/pkgs/slang/slang-2.3.1a.tar.bz2/sha512/e7236a189081ebcbaf4e7f0506671226a4d46aede8826e1a558f1a2f57bcbe3ad58eadeabe2df99cd3d8bacb4c93749996bcbce4f51d338fc9396e0f945933e7/slang-2.3.1a.tar.bz2"
```

Comment:
again I don't favor removing hard to find mirror links over this line number thing. I think we need to just set a higher number.

Reply:
I can add things to ignore RuboCop checks on these lines if that's more desired. I don't want to leave the global value to the hilarious one that's currently required because Fedora likes URLs the length of a small town.

Reply:
is there no way to tell rubcop to ignore url lines? this seems really lame

Reply:
Yes, we could ignore all url/mirror lines. That may be a better option.

Reply:
:+1: